app.py

```diff
@@ -1,7 +1,141 @@
-from flask import Flask, render_template
-app = Flask(__name__)
+"""
+A Flask application for snapcraft.io.
+
+The web frontend for the snap store.
+"""
+
+import flask
+import requests
+import requests_cache
+import datetime
+import json
+import humanize
+from dateutil import parser
+
+
+app = flask.Flask(__name__)
+uncached_session = requests.Session()
+cached_session = requests_cache.CachedSession(expire_after=60)
+request_timeout = 2
+snap_details_url = (
+    "https://api.snapcraft.io/api/v1/snaps/details/{snap_name}"
+    "?channel=stable"
```

Comment:
FWIW `?channel=stable` is the default scope and because the way `channel_maps_list` field is exposed (all channels), it's unlikely that the store-front would ever need to switch scope.

Reply:
I was aware it's the default, but I'd still rather be explicit in this URL. It informs the reader of this code immediately (without having to have knowledge of the API) that we're only requesting stable channels, and illustrated how to filter queries in general.